week-11-homework-react-math-calc/src/Calculator.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+export default class Calculator extends React.Component {
+
+        constructor () {
+            super ()
+
+            this.state = {
+                firstOperand: 0,
+                operator: '+',
+                secondOperand: 0,
+                result: '0'
+            }
+            //this.onFirstOperandChange = this.onFirstOperandChange.bind(this)
+
+        }
+
+
+        onFirstOperandChange = (event) => {
+            const newValue = event.target.value;
+            this.setState({
+                firstOperand: newValue
+            });
+        }
+
+        onSecondOperandChange = (event) => {
+            const secondOperand = event.target.value;
+            this.setState({
+                secondOperand
+            });
+         }
+
+         onOperandChange = (event) => {
+             const operator = event.target.value;
+                this.setState({
+                    operator
+                })
+            
+        
+         }
+
+         calculate=() => {
+            let result = 0;
+
+            const {firstOperand, secondOperand, operator} = this.state;
+
+            switch (operator) {
+                case '+':
+                    result = parseInt(firstOperand) + parseInt(secondOperand);
+                    break;
+                case '-':
+                    result = parseInt(firstOperand) - parseInt(secondOperand);
+                    break;
+                case '*':
+                    result = parseInt(firstOperand) * parseInt(secondOperand);
+                    break;
+                case '/':
+                    result = parseInt(firstOperand) / parseInt(secondOperand);
+                    break;
+            }
+            this.setState({
+                result
+            })
+         }
+
+    render() {
+        return (
+            <div>
+
+                {/* first operand */}
+                < input type="number" onChange={this.onFirstOperandChange} />
+
+                <select onChange={this.onOperandChange} >
+                    <option>*</option>
+                    <option>/</option>
+                    <option>+</option>
+                    <option>-</option>
+                </select>
+
+                {/* second operand */}
+                < input type="number" onChange={this.onSecondOperandChange}/>
+                
+                <button onClick={this.calculate}>=</button>
+
+                <input value={this.state.result} type="text" disabled />
+
+
+                
+
+
+
+
```

Comment:
There are a couple spots like this where you have a lot of white space that you shouldn't have.  You want the code to look clean and adding this extra white space breaks that up.